### PR TITLE
Add Images

### PR DESCRIPTION
**Pull request by gferioli0418**
-Add bootstrap 4 to use some styling
-Add Images tab to show images of me with small summaries
-Add a link to go to Images tab in the main page.

### CODE REVIEW

**Comment by gferioli0418** [comment]
For the Add blogs page:
-Add a list of blogs that have my name
-update the main page to go to the blogs page
-update link to go to linkedin

**Comment by gferioli0418** [comment]
For add Project and resume:
-Include resume link
-add an under-construction project page
-add libraries for the projects(P5.js)

**Comment by gferioli0418** [comment]
Finished adding projects:
-take out p5 library locally and add a remote P5 CDN.
-Finish adding projects.
-Include a project page.
-remove SanJuan1a.jpg

File: portfolio/src/main/webapp/Projects/Projects.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			</div>
+		</div>
+		<div class="column">
+			<div class="content">
+				<a href="https://github.com/gferioli0418/Analyzing-Mobile-App-Data">
+					<img src="data.jpg" style="width:100%">
+                </a>
+
+					<h3>Google play store and App Store analysis</h3>
+					<p>First project using Python. This project was mainly about analysing data. Specifically, I anaylze
```
Comment (josueetcom) [inline portfolio/src/main/webapp/Projects/Projects.html]: *analyze

File: portfolio/src/main/webapp/Projects/Projects.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			</div>
+		</div>
+		<div class="column">
+			<div class="content">
+				<a href="snakeProject/snakeProject.html">
+					<img src="snake.jpg" style="width:100%">
+                </a>
+
+					<h3>Snake game</h3>
+					<p>The typicall snake game. Everyone knows what this is. And if you dont know ... well.. I dont
```
Comment (josueetcom) [inline portfolio/src/main/webapp/Projects/Projects.html]: *typical

File: portfolio/src/main/webapp/Projects/ballProject/ballProject.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0>
+	<style>
+		body {
+			padding: 0;
+			margin: 0;
+		}
+	</style>
+	<script src="https://cdn.jsdelivr.net/npm/p5@0.10.2/lib/p5.js"></script>
```
Comment (josueetcom) [inline portfolio/src/main/webapp/Projects/ballProject/ballProject.html]: Sweet!

File: portfolio/src/main/webapp/Projects/snakeProject/sketch.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  if (s.eat(food)) {
+    foodLocation();
+  }
+  frameRate(10);
+  fill(153, 42, 99);
+  rect(food.x, food.y, scl, scl);
+}
+function keyPressed() {
+  if (keyCode === UP_ARROW && up) {
+    s.direction(0, -1);
```
Comment (josueetcom) [inline portfolio/src/main/webapp/Projects/snakeProject/sketch.js]: Optional: I know this isn't strictly part of the SPS project but you could consider using something [enum-like](https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/) here.

I could see this code being transformed from:
```javascript
var up = true;
var down = true;
var left = true;
var right = true;
...
if (keyCode == UP_ARROW && up) {
  s.direction(0, -1);
  up = true;
  down = false;
  left = true;
  right = true;
} ...
```
to something like:
```javascript
const directions = {
    UP: 'up',
    DOWN: 'down',
    LEFT: 'left',
    RIGHT: 'right'
}
var dir = null;
...
if (keyCode == UP_ARROW && dir != directions.DOWN) {
  dir = directions.UP;
  s.direction(dir);
} ...
```

Again, this is completely optional. It's a cool project regardless!

**Comment by gferioli0418** [comment]
Fix some grammar Errors:
-include Project ID to be able to host a live server
-Fix some grammar errors there were in the project page

**Pull request merged**